tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = 
+  py36
+
+[testenv]
+deps = 
```

Comment:
Let's make the only tox dep `pipenv==2018.7.1` and instead add `pipenv sync -d` as a first command in `commands`; this way the dependency information is consolidated in the Pipfile and we don't repeat it here, instead we use pipenv to install the dev dependencies in each toxenv.